docker-jans-shibboleth/scripts/shib_setup.py

```
@@ -0,0 +1,235 @@
+import os
+import shutil
+from pathlib import Path
+from string import Template
+
+logger = logging.getLogger("shibboleth")
+
+SHIBBOLETH_HOME = os.environ.get("SHIBBOLETH_HOME", "/opt/shibboleth-idp")
+
+
+class ShibbolethSetup:
+    def __init__(self, manager):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)